Task1/weight.rb

```diff
@@ -0,0 +1,21 @@
+#ideal weight
+MARGIN = 110
+class Ideal
+  def initialize
+    puts "Enter your name: "
+    @name = gets.chomp.capitalize
+    puts "Enter your height: "
+    @height = gets.chomp.to_i
+  end
+
```

Comment:
add validations to user input
you can also print an error if the user passes a negative height